react-components/toolbar-filter/step6.md

```diff
@@ -1,95 +1,13 @@
-One particular type of Data toolbar item is called the `DataToolbarFilter`. When used, selected filters are displayed in chip groups below the toolbar. You may consider placing any `DataToolbarFilter` inside of a toggle group to make your toolbar responsive.
+Managing the relationship of a toolbar with its filters to a table or other component is the responsibility of the consumer. 
```

Comment:
Who do we mean by consumer?

Reply:
We mean the person trying to use the component in their code. Not the end user.

Reply:
I believe the user/developer building the toolbar.